packages/context-editor/src/components/AttributePanel.tsx

```diff
@@ -41,19 +42,22 @@ export function AttributePanel({ menuHidden }: { menuHidden: boolean }) {
 		setPosition: setActiveNodePosition,
 	} = useEditorContext();
 
-	/**
-	 * This determination of the 'activeNode' is prone to bugs. We should figure
-	 * out a better way to do it.
-	 **/
```

Comment:
it's still a bit more bespoke than I'd like, but at least there are tests for it now so should hold up okay 🤞